lib/controllers/playlistsController.js

```diff
@@ -4,6 +4,22 @@ const index = (request, response) => {
 }
 
 const create = async (req, res) => {
+  if (req.body.title) {
+    Playlist.create({ title: req.body.title })
+      .then(playlist => {
+        res.status(201).json({ id: playlist[0].id,
+                            title: playlist[0].title,
+                        createdAt: playlist[0].created_at,
+                        updatedAt: playlist[0].updated_at
+                        });
+      })
```

Comment:
Looks good. I made a helper method, `formatPayload` that you should be able to use here. Its usage should look something like this: `res.status(201).json(formatPayload(playlist[0]))` Which is pretty must that same as line 59 in the file.

Reply:
Changes have been made, thanks for adding that helper method.